spring-data-vmware-gemfire-acceptanceTests/src/test/java/org/springframework/context/ApplicationContextBeanOrderingIntegrationTests.java

```diff
@@ -98,7 +96,7 @@ public void beansAreOrderedByBeanDefinitionDeclarationOrder() {
 			.map(Object::toString)
 			.collect(Collectors.toList());
 
-		assertThat(beanNames)
+		Assertions.assertThat(beanNames)
```

Comment:
IJ seems to like to change this import and add `Assertions` where it isn't required when moving classes. I had to fix this kind of thing several times while doing this work.